apis/apps/v1alpha1/componentdefinition_types.go

```diff
@@ -509,6 +509,7 @@ const (
 	OfficialPostgresqlBuiltinActionHandler BuiltinActionHandlerType = "official-postgresql"
 	ApeCloudPostgresqlBuiltinActionHandler BuiltinActionHandlerType = "apecloud-postgresql"
 	PolarDBXBuiltinActionHandler           BuiltinActionHandlerType = "polardbx"
+	CustomActionHandler                    BuiltinActionHandlerType = "custom"
```

Comment:
custom is not builtin

Reply:
custom handler is utilized to provide support for executing self-defined commands when there is no built-in handler available.